data/drop-tables.js

```diff
@@ -0,0 +1,21 @@
+const pool = require('../lib/utils/pool');
+
+run();
+
+async function run() {
+
+  try {
+    await pool.connect();
+
+    await pool.query(`
+        DROP TABLE IF EXISTS links CASCADE;
```

Comment:
Same question here as Feature/20. Is CASCADE needed if the tables are not joined?